pkgs/tools/graphics/grace/default.nix

```diff
@@ -0,0 +1,74 @@
+{ lib
+, gccStdenv
+, fetchurl
+, coreutils
+, motif
+, fftw
+, netcdf
+, libpng
+, libjpeg_original
+, libX11
+, libSM
+, libXext
+, libXmu
+, libXp
+, libXpm
+, libXt
+}:
+
+gccStdenv.mkDerivation rec {
```

Comment:
```suggestion
stdenv.mkDerivation rec {
```

Reply:
Hi,

I used gccStdenv as clang failed to built but is the default on Darwin systems.